src/main/java/org/broadinstitute/hellbender/tools/spark/sv/utils/SVFileUtils.java

```diff
@@ -0,0 +1,184 @@
+package org.broadinstitute.hellbender.tools.spark.sv.utils;
+
+import htsjdk.samtools.*;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.tools.spark.utils.HopscotchSet;
+import org.broadinstitute.hellbender.utils.Utils;
+import org.broadinstitute.hellbender.utils.gcs.BucketUtils;
+import org.broadinstitute.hellbender.utils.io.IOUtils;
+
+import java.io.*;
+import java.util.*;
+
+public final class SVFileUtils {
+
+    private static final String REFERENCE_GAP_INTERVAL_FILE_COMMENT_LINE_PROMPT = "#";
+
+    /**
+     * Creates a directory, in local FS, HDFS, or Google buckets to write to.
+     */
+    public static boolean createDirToWriteTo(final String pathString) {
```

Comment:
I think that this method, with some refactoring, could belong to `IOUtils`.

Reply:
Can you elaborate, specifically what refactoring is needed?

Reply:
I think that it would work by using the following code (but I haven't test it):

```java
// this should take care of GCS
final Path dir = IOUtils.getPath(Utils.nonNull(pathString));
if (Files.exists(dir)) {
    throw new IOException("Directory to be created already exists: " + dir.toUri());
}
// maybe some exceptions should be catch to return false (I am not sure about the usage of the boolean)
Files.createDirectory(dir);
return Files.isDirectory(dir) & Files.isWritable(dir);
```

In that case, it belongs to `IOUtils`, because it can be re-used easily...

Reply:
I've done mostly what you suggested, can you take a look again?